src/interfaces/IArrangerConduit.sol

```diff
@@ -0,0 +1,260 @@
+     */
+    function totalInterestEarned(address asset)
+        external view returns (uint256 totalInterestEarned_);
+
+    /**
+     *  @dev    Returns the total positions for a given asset.
+     *  @param  asset            The address of the asset.
+     *  @return totalPositions_ The total positions held in the asset.
+     */
+    function totalPositions(address asset) external view returns (uint256 totalPositions_);
+
+    /**
+     *  @dev    Returns the total amount that can be withdrawn for a given asset.
+     *  @param  asset              The address of the asset.
+     *  @return totalWithdrawable_ The total amount that can be withdrawn from the asset.
+     */
+    function totalWithdrawable(address asset) external view returns (uint256 totalWithdrawable_);
+
+    /**
+     *  @dev    Returns the details of a specific fund request.
+     *  @param  asset         The address of the asset.
+     *  @param  fundRequestId The ID of the fund request.
+     *  @return fundRequest_  The details of the fund request.
+     */
+    // TODO: Figure out how to get this to compile
+    // function fundRequests(address asset, uint256 fundRequestId)
+    //     external view returns (FundRequest[] memory fundRequest_);
+
+    /**
+     *  @dev    Returns the pending withdrawals for a given asset and allocator.
+     *  @param  ilk                 The unique identifier of the for a particular ilk.
+     *  @param  asset               The address of the asset.
+     *  @return pendingWithdrawals_ The amount of pending withdrawals for the allocator.
+     */
+    function pendingWithdrawals(bytes32 ilk, address asset)
+        external view returns (uint256 pendingWithdrawals_);
+
+    /**
+     *  @dev    Returns the position for a given ilk and asset.
+     *  @param  ilk        The unique identifier of the for a particular ilk.
+     *  @param  asset      The address of the asset.
+     *  @return positions_ The positions for the given ilk and asset.
+     */
+    function positions(bytes32 ilk, address asset) external view returns (uint256 positions_);
+
+    /**********************************************************************************************/
+    /*** Router Functions                                                                       ***/
+    /**********************************************************************************************/
+
+    /**
+     *  @dev    Function to initiate a withdrawal request from a Fund Manager.
+     *  @param  ilk           The unique identifier of the for a particular ilk.
+     *  @param  asset         The asset to withdraw.
+     *  @param  amount        The amount of tokens to withdraw.
+     *  @param  data          Arbitrary encoded data to provide additional info to the Fund Manager.
+     *  @return fundRequestId The ID of the withdrawal request.
+     */
+    function requestFunds(bytes32 ilk, address asset, uint256 amount, bytes memory data)
```

Comment:
We can drop the `data` argument now that this is no longer a generalized interface.